pulp_container/app/tasks/sync_stages.py

```diff
@@ -176,6 +170,23 @@ async def run(self):
 
                 digest = response.artifact_attributes["sha256"]
 
+                # Look for cosign signatures
+                # cosign signature has a tag convention 'sha256-1234.sig'
+                if self.signed_only and not signature_source:
+                    if (
+                        not (tag_name.endswith(".sig") and tag_name.startswith("sha256-"))
+                        and f"sha256-{digest}.sig" not in tag_list
+                    ):
+                        # skip this tag, there is no corresponding signature
+                        log.info(
```

Comment:
This is behavioral change, we no longer raise an error but skip and log.

Reply:
Can we now remove the second check?
```
                            if self.signed_only and not man_sig_dcs:
```
Or, perhaps, reassess the need for its break statement?